code/modules/awaymissions/mission_code/challenge.dm

```diff
@@ -19,7 +19,7 @@
 	name = "Energy Cannon"
 	desc = "A heavy duty industrial laser."
 	icon = 'icons/obj/singularity.dmi'
-	icon_state = "emitter"
+	icon_state = "emitter_+a"
```

Comment:
fixed this obj's icon, since it doesn't require a powernet and is only used at the CTF area.